### DISCUSSION TITLE
Avoid repeating the header verb

### DISCUSSION THREAD

**Discussion by contivero** [olivierlacan/keep-a-changelog#474] [Ideas]
Branching from the discussion in #54 (particularly [this comment](https://github.com/olivierlacan/keep-a-changelog/issues/54#issuecomment-369208269)).
The proposal is doing, e.g.:
```
#Added
 - support for x.
 - new logo image for y.
```
instead of repeating the verb in each entry, as in:
```
#Added
 - added support for x.
 - added a new logo image for y.
```
since doing so is clearly redundant and unnecessary noise.

Also, considering doing so reads like a single sentence, I'd argue against capitalizing the starting word of each entry. Following the example, not writing "Support" nor "New", but "support" and "new", since we are effectively reading "Added support...", "Added new...".

Would love to hear opinions on the issue.

**Comment by aryelgois** [comment]
I usually add a verb only in Changed and Fixed to tell how it happened, but not in the others since it is obvious :)

I would consider not capitalizing the starting word, unless required.

I also have a personal preference: not ending each list entry (i.e. with a dot, semicolon or anything)
- only dots look odd to me.
- semicolons make it not look like a list, but like code();
- sometimes, comma is used in the entry,
- mixing them makes a ugly commit history changing the dot in last entry `.` `;`
- just to append something.

> it was weird to write that 

- instead of that
- just embrace
- YAML lists
- (and whoever also does lists like this :D )

Also, do you have a preference over `-`, `+` or `*` to write lists?  
In a compact keyboard, `-` is only one key, the others need `shift`
